.agent/workflows/commit.md

```
@@ -1,10 +1,10 @@
-// turbo-all
-
-1. `git status` を実行して、ステージングされていない変更や新規ファイルがないか確認します。
-2. 必要に応じて `git add .` または個別のファイルを追加してステージングします。
-3. `git diff --cached` を実行して、これからコミットされる内容（差分）を最終確認します。
-4. 変更内容に適したコミットメッセージを作成し、`git commit -m "..."` を実行してよいか、ユーザーに確認します。
\ No newline at end of file
+---
+description: 変更をコミットする（git status, git add）
+---
+
+// turbo-all
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)